package.json

```diff
@@ -4,12 +4,15 @@
   "description": "",
   "main": "",
   "scripts": {
-    "lint": "tsqllint"
```

Comment:
`cross-env-shell` allows us to use the `$INIT_CWD` environment variable on all runners (windows included).

The reason why we needed to `cd $INIT_CWD` is mentioned in the [npm run documentation](https://docs.npmjs.com/cli/v7/commands/npm-run-script#description). For ease, here's the applicable excert:

> Scripts are run from the root of the package folder, regardless of what the current working directory is when npm run is called. If you want your script to use different behavior based on what subdirectory you're in, you can use the INIT_CWD environment variable, which holds the full path you were in when you ran npm run.